### PR TITLE
Bsnl prod branch terraform and github action changes

### PR DESCRIPTION
**Pull request by ppratikcr7**

### CODE REVIEW

**Comment by ppratikcr7** [comment]
> why are we overwriting all of the CL dev environment configs? is this just a temporary setup?

Hey Dan, dont worry, its just temporary changes for our BSNL deployment, I wanted to show the changes made to our team. This PR is not to be reviewed or merged. :)

File: backend/terraform/environments/production/tfvars.sample [hunk 1]
```diff
@@ -31,8 +31,8 @@ APP_DEMO                = false
...
-METRIC                  = "[{'metric': 'totalTimeSeconds','datatype': 'continuous'}, { 'groupClass': 'masteryWorkspace', 'allowedKeys': [ 'calculating_area_various_figures', 'Compare_functions_diff_reps_quadratic' ], 'attributes': [{ 'metric': 'timeSeconds', 'datatype': 'continuous'}]}]"
+CONTEXT_METADATA        = "{\"add\":{\"EXP_IDS\":[\"add-id1\",\"add-id2\"],\"EXP_POINTS\":[\"add-point1\",\"add-point2\"],\"GROUP_TYPES\":[\"add-group1\",\"add-group2\"],\"CONDITIONS\":[\"add-con1\",\"add-con2\",\"add-con3\"]},\"sub\":{\"EXP_IDS\":[\"sub-id1\",\"sub-id2\"],\"EXP_POINTS\":[\"sub-point1\",\"sub-point2\"],\"GROUP_TYPES\":[\"sub-group1\",\"sub-group2\"],\"CONDITIONS\":[\"sub-con1\",\"sub-con2\",\"sub-con3\"]},\"mul\":{\"EXP_IDS\":[\"mul-id1\",\"mul-id2\"],\"EXP_POINTS\":[\"mul-point1\",\"mul-point2\"],\"GROUP_TYPES\":[\"mul-group1\",\"mul-group2\"],\"CONDITIONS\":[\"mul-con1\",\"mul-con2\",\"mul-con3\"]},\"div\":{\"EXP_IDS\":[\"div-id1\",\"div-id2\"],\"EXP_POINTS\":[\"div-point1\",\"div-point2\"],\"GROUP_TYPES\":[\"div-group1\",\"div-group2\"],\"CONDITIONS\":[\"div-con1\",\"div-con2\",\"div-con3\"]}}"
```
Comment (VivekFitkariwala) [inline backend/terraform/environments/production/tfvars.sample:34]: We should have this change in main

File: frontend/terraform/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+### Deployment Overview
```
Comment (VivekFitkariwala) [inline frontend/terraform/README.md:1]: We should also add this change in main

**Comment by danoswaltCL** [comment]
is this still needed? can it be removed as a PR now?

**Comment by ppratikcr7** [comment]
> is this still needed? can it be removed as a PR now?

Hey Dan, can we still have this for some time, I wont transfer some of the above changes for our main CL repo dev branch. I will close this once done. Thanks :)

**Comment by danoswaltCL** [comment]
yeah of course, just wondered.

**Pull request closed without merging**